egs/wsj/s5/steps/chaina/internal/get_train_schedule.py

```diff
@@ -52,6 +51,9 @@ def get_args():
                         (to cover frame-shifted copies of the data), times
                         the value of --num-repeats given to process_egs.sh,
                         times any factor arising from data augmentation.""")
+    parser.add_argument("--num-repeats", type=float, default=1.0,
+                        help="""The number of repeats...TODO
+                        .""")
```

Comment:
Don't add this arg... just use frame_subsampling_factor as the number of repeats.